tests/backup/backup_sse_test.go

```diff
@@ -0,0 +1,412 @@
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", newBackupLocationWithSseAfterRestart))
+				backupLocationMap[backupLocationUID] = newBackupLocationWithSseAfterRestart
+				log.Infof("created backup location successfully")
+			})
+			Step("Taking backup of application with new BackupLocation post px-backup and stork restart", func() {
+				log.InfoD("Taking backup of application with new BackupLocation post px-backup and stork restart")
+				backupNameAfterPxBackupRestart = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, bkpNamespaces[0], time.Now().Unix())
+				appContextsToBackup = FilterAppContextsByNamespace(scheduledAppContexts, []string{bkpNamespaces[0]})
+				err = CreateBackupWithValidation(ctx, backupNameAfterPxBackupRestart, SourceClusterName, newBackupLocationWithSseAfterRestart, backupLocationUID, appContextsToBackup, make(map[string]string), orgID, clusterUid, "", "", "", "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup with new BackupLocation post px-backup and stork restart [%s]", backupNameAfterPxBackupRestart))
+			})
+			Step("Create restore with backup taken on new BackupLocation created post px-backup and stork restart", func() {
+				log.InfoD("Create restore with backup taken on new BackupLocation created post px-backup and stork restart")
+				restoreName := fmt.Sprintf("restore-with-replace-from-new-bl-%s-%v", RestoreNamePrefix, time.Now().Unix())
+				err = CreateRestoreWithReplacePolicy(restoreName, backupNameAfterPxBackupRestart, make(map[string]string), SourceClusterName, orgID, ctx, make(map[string]string), 2)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore [%s]", restoreName))
+				restoreList = append(restoreList, restoreName)
+			})
+			Step("Remove deny policy from S3 bucket which is used in customBackupLocationWithSse", func() {
+				log.InfoD("Remove deny policy from S3 bucket")
+				err = RemoveS3BucketPolicy(customBucketsWithDenyPolicy[1])
+				dash.VerifySafely(err, nil, fmt.Sprintf("Verify removal of deny policy from s3 bucket"))
+			})
+			Step("Taking backup of application after removal of deny policy and sse still set to true", func() {
+				log.InfoD("Taking backup of application after removal of deny policy and sse still set to true")
+				backupNameAfterRemovalOfDenyPolicy = fmt.Sprintf("%s-%s-%v", BackupNamePrefix, bkpNamespaces[0], time.Now().Unix())
+				appContextsToBackup = FilterAppContextsByNamespace(scheduledAppContexts, []string{bkpNamespaces[0]})
+				err = CreateBackupWithValidation(ctx, backupNameAfterRemovalOfDenyPolicy, SourceClusterName, newBackupLocationWithSseAfterRestart, backupLocationUID, appContextsToBackup, make(map[string]string), orgID, clusterUid, "", "", "", "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of backup after removal of deny policy and sse still set to true [%s]", backupNameAfterRemovalOfDenyPolicy))
```

Comment:
Is this expected to fail ? What is the expected error that we need to compare here in that case ?